sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/text/TextSuite.scala

```diff
@@ -17,9 +17,11 @@
 
 package org.apache.spark.sql.execution.datasources.text
 
+import com.google.common.io.Files
+import org.apache.hadoop.io.compress.GzipCodec
 import org.apache.spark.sql.test.SharedSQLContext
 import org.apache.spark.sql.types.{StringType, StructType}
-import org.apache.spark.sql.{AnalysisException, DataFrame, QueryTest, Row}
+import org.apache.spark.sql._
```

Comment:
Sorry if done by plugin, it's correct. Did you use this plugin?
https://plugins.jetbrains.com/plugin/7350